recipes-core/packagegroups/packagegroup-ni-internal-deps.bb

```diff
@@ -87,3 +87,10 @@ RDEPENDS:${PN} += "\
 	make \
 	pkgconfig \
 "
+
+# Required by PXIPS ATS events
+# Team: PXI PS
+# Contact: Kevin Khai-Wern Lim
+RDEPENDS:${PN} += "\
+	memtester \
```

Comment:
Do we need to remove `memtester` from [packagefeed-ni-extra.bb](https://github.com/ni/meta-nilrt/blob/7ffc454b3fab7c3ee5c4d3aff119a0751f6e63d5/recipes-core/packagegroups/packagefeed-ni-extra.bb#L235) since we're adding it here?

Reply:
No. It should be automatically deconflicted by the build scripting.

Reply:
I get that the build tooling can de-conflict it but from a maintenance perspective it seems odd to me to have it both in `packagefeed-ni-extra.bb` and `packagefeed-ni-core.bb` (via packagegroup-ni-internal-deps). In the past IIRC when we moved things into internal deps we removed it from extra.

Reply:
You're right that people do that, but it has never been necessary. I prefer to be conservative wrt. ensuring that needed deps get built into **a** feed, and avoid the possibility that something gets accidentally dropped from all he feeds.